website/content/donate.njk

```diff
@@ -56,79 +56,54 @@ title: Donate
-                {% if patreon.tier == "EnvironmentMapEffect" %}
-                <div class="col-sm-4 mb-1 text-center">
-                    {% if patreon.url %}<a href="{{patreon.url}}">{% endif %}
-                        {{patreon.name}}
-                    {% if patreon.url %}</a>{% endif %}
-                    <br />
-                </div>
-                {% endif %}
-            {% endfor %}
-        </div>
-        <br />
-        <h4 class="text-center" id="Model.Draw">Model.Draw</h4>
-        <div class="row justify-content-center mb-3">
-            {% for patreon in patreons | sort() %}
-                {% if patreon.tier == "Model.Draw" %}
-                <div class="col-sm-4 mb-1 text-center">
-                    {% if patreon.url %}<a href="{{patreon.url}}">{% endif %}
-                        {{patreon.name}}
-                    {% if patreon.url %}</a>{% endif %}
-                    <br />
-                </div>
-                {% endif %}
-            {% endfor %}
-        </div>
         <br />
-        <h4 class="text-center" id="DrawUserPrimitives">DrawUserPrimitives</h4>
-        <div class="row justify-content-center mb-3">
-            {% for patreon in patreons | sort() %}
-                {% if patreon.tier == "DrawUserPrimitives" %}
-                <div class="col-sm-4 mb-1 text-center">
-                    {% if patreon.url %}<a href="{{patreon.url}}">{% endif %}
-                        {{patreon.name}}
-                    {% if patreon.url %}</a>{% endif %}
-                    <br />
-                </div>
-                {% endif %}
-            {% endfor %}
-        </div>
-        <br />
-        <h4 class="text-center" id="SpriteBatch">SpriteBatch</h4>
-        <div class="row justify-content-center mb-3">
-            {% for patreon in patreons | sort() %}
-                {% if patreon.tier == "SpriteBatch" %}
-                <div class="col-sm-4 mb-1 text-center">
-                    {% if patreon.url %}<a href="{{patreon.url}}">{% endif %}
-                        {{patreon.name}}
-                    {% if patreon.url %}</a>{% endif %}
-                    <br />
-                </div>
-                {% endif %}
-            {% endfor %}
-        </div>
-        <br />
-        <h4 class="text-center" id="Cornflower Blue">Cornflower Blue</h4>
-        <div class="row justify-content-center mb-3">
-            {% for patreon in patreons | sort() %}
-                {% if patreon.tier == "Cornflower Blue" %}
-                <div class="col-sm-4 mb-1 text-center">
-                    {% if patreon.url %}<a href="{{patreon.url}}">{% endif %}
-                        {{patreon.name}}
-                    {% if patreon.url %}</a>{% endif %}
-                    <br />
-                </div>
-                {% endif %}
-            {% endfor %}
+        
+        <!-- Table Structure -->
+        <div class="table-responsive">
+            <table class="table table-bordered text-center">
```

Comment:
So this stuff...

```
<div class="row justify-content-center mb-3">
```

Is how tables are made in Bootstrap.  They call them "grids" but basically they are tables.

Here are the docs on these:

https://getbootstrap.com/docs/4.0/layout/grid/

I think it would be better to stick to this instead of going old school with `<table>`.

Reply:
Done!